react/ui-components/src/hoc/InboxSearchComposer.js

```
@@ -113,13 +141,14 @@ const InboxSearchComposer = ({configs,headerLabel,additionalConfig,onFormValueCh
     
-    const updatedReqCriteria = Digit?.Customizations?.[apiDetails?.masterName]?.[apiDetails?.moduleName]?.preProcess ? Digit?.Customizations?.[apiDetails?.masterName]?.[apiDetails?.moduleName]?.preProcess(requestCriteria,configs.additionalDetails) : requestCriteria 
+    const configModule = Digit?.Customizations?.[apiDetails?.masterName]?.[apiDetails?.moduleName]
+    const updatedReqCriteria = configModule?.preProcess ? configModule?.preProcess(requestCriteria,configs.additionalDetails) : requestCriteria 
     
-    if(configs.customHookName){
+    if(configs?.customHookName){
         var { isLoading, data, revalidate,isFetching,refetch,error } = eval(`Digit.Hooks.${configs.customHookName}(updatedReqCriteria)`);
     }
     else {
-       var { isLoading, data, revalidate,isFetching,error } = Digit.Hooks.useCustomAPIHook(updatedReqCriteria);
+       var { isLoading, data, revalidate,isFetching,error,refetch } = Digit.Hooks.useCustomAPIHook(updatedReqCriteria);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'isLoading'. Consider to delete it or rename it.

'isLoading' is defined here:


(lint/suspicious/noRedeclare)